modules/worker/create-replication-data-worker.js

```diff
@@ -70,14 +70,13 @@ process.on('message', async (data) => {
                 distributionEpk: distEpk,
             };
 
-            writeFilePromises.push(Utilities.writeContentsToFile(cacheDirectoryPath, `${color}.json`, JSON.stringify(replication)));
-
+            const fullPath = path.join(cacheDirectoryPath, `${color}.json`);
+            fs.writeFileSync(fullPath, JSON.stringify(replication));
 
             hashes[`${color}LitigationHash`] = litRootHash;
             hashes[`${color}DistributionHash`] = distRootHash;
         }
 
-        await Promise.all(writeFilePromises);
```

Comment:
We should also remove the declaration of writeFilePromises

Reply:
good catch